### DISCUSSION TITLE
4連ピンソケットの破損について

### DISCUSSION THREAD

**Discussion by conduc1978** [Yowkees/keyball#318] [General]
OLEDの4連ピンヘッダをピンソケットに挿入する際に折ってしまいました。
部品は注文できたのですが、ピンソケットを取ることができません。
おそらく半田が内部に入りこんでしまったのだと思います。

どのように外せば良いかご教授ください。
お願いします。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

4連ピンソケットは4ピンとも同時に熱を加えると軽い力で引き抜くことができます。
具体的には半田吸い取り線を４ピンの幅に切り、ピンに沿って半田付けします。その後に吸い取り線を加熱するとすべての半田が溶けて除去できます。
難しい場合には、ピンソケットをバラバラに壊す覚悟で１ピンずつ加熱して引き抜く方も見えます。
新しい4PピンソケットはKeyballシリーズご購入者には部品で販売可能ですので、ご希望の場合にはご連絡ください。

以上、ご検討お願いいたします。

**Reply by conduc1978** [reply]
ありがとうございます。

結局ピンソケットをバラバラにして外すことができました。
ですが、基盤が壊れてしまったのではないかと不安があります。
写真を載せますので、ご判断いただければと思います。
どうぞよろしくお願いします。
![IMG_2189](https://github.com/Yowkees/keyball/assets/142858574/41fb3c51-f724-42c8-b9c3-bbb8314aab7a)
![IMG_2191](https://github.com/Yowkees/keyball/assets/142858574/72ccf830-a3de-4361-87c3-ee937f2cd505)

**Reply by Yowkees** [reply]
半田付けが可能なようであれば大丈夫だと思います。

もしミドル基板が必要でしたら、Keyballシリーズ購入者様向けに部品単位での販売もありますので、白銀ラボウェブショップよりお問合せいただければ幸いです。

以上、お手数ですがご確認お願いいたします。

**Reply by conduc1978** [reply]
ありがとうございます！
半田付けをしたら無事にできました。
ようやく完成して、これから楽しんでいきたいと思います。
本当にありがとうございました。

**Comment by Yowkees** [comment]
無事に半田付けできましたか！動いてよかったです。
ぜひ末永くご愛用いただければ幸いです。
